demisto_sdk/commands/changelog/changelog_obj.py

```diff
@@ -0,0 +1,57 @@
+from enum import Enum
+from typing import List, Tuple
+
+from pydantic import BaseModel, validator
+
+INITIAL_LOG = {
+    "logs": [
+        {
+            "description": "enter description about this PR",
+            "type": "<fix|feature|breaking>",
+        }
+    ]
+}
+
+class logType(str, Enum):
```

Comment:
```suggestion
class LogType(str, Enum):
```
change elsewhere as well